sdk/storage/azure-storage-queue/src/main/java/com/azure/storage/queue/implementation/util/BuilderHelper.java

```diff
@@ -147,8 +147,9 @@ public static QueueUrlParts parseEndpoint(String endpoint, ClientLogger logger)
      * @param logger {@link ClientLogger} used to log any exception.
      * @return A new {@link HttpPipeline} from the passed values.
      */
-    public static HttpPipeline buildPipeline(StorageSharedKeyCredential storageSharedKeyCredential,
-        TokenCredential tokenCredential, SasTokenCredential sasTokenCredential, String endpoint,
+    public static HttpPipeline buildPipeline(
+        StorageSharedKeyCredential storageSharedKeyCredential,
+        TokenCredential tokenCredential, AzureSasCredential sasTokenCredential, String endpoint,
```

Comment:
nit: update the param name to match the type - `azureSasCredential`
```suggestion
        TokenCredential tokenCredential, AzureSasCredential azureSasCredential, String endpoint,
```

Reply:
done. I did some broader renaming here and there as well.